services/src/main/java/org/keycloak/services/clientpolicy/context/TokenResponseContext.java

```diff
@@ -33,15 +34,18 @@ public class TokenResponseContext implements ClientPolicyContext {
     private final OAuth2CodeParser.ParseResult parseResult;
     private final ClientSessionContext clientSessionCtx;
     private final TokenManager.AccessTokenResponseBuilder accessTokenResponseBuilder;
+    private AccessTokenResponse accessTokenResponse;
 
     public TokenResponseContext(MultivaluedMap<String, String> params,
             OAuth2CodeParser.ParseResult parseResult,
             ClientSessionContext clientSessionCtx,
-            TokenManager.AccessTokenResponseBuilder accessTokenResponseBuilder) {
+            TokenManager.AccessTokenResponseBuilder accessTokenResponseBuilder,
```

Comment:
As in my previous comment, isn't the builder enough?

Reply:
The same comment just above.